Gemfile

```diff
@@ -5,4 +5,5 @@ gemspec
 
 group :test, :development do
   gem "byebug", :platforms => [:mri_20, :mri_21]
+  gem "mocha"
```

Comment:
Mainly as a convenience for [cases like this](https://github.com/github/github-ldap/pull/94/files#diff-c793d1b8aa9fb622f67032b4b0f7020dR136).